book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/1_0_20_duckiebot_SD_initialization.md

```diff
@@ -49,6 +49,8 @@ The basic options are:
                        the flashed device The default is: quackquack
```

Comment:
We need to remove the username and password from here (and, @afdaniele also from dts itself, right?). And we need to mention that they will be the default.

Reply:
I can just move them to the non-essential commands. Or, are they not going to be valid commands anymore?